core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -367,21 +367,12 @@ trait WhiskActionsApi extends WhiskCollectionAPI with SequenceActions with Refer
                 l.memory getOrElse MemoryLimit(),
                 l.logs getOrElse LogLimit())
         } getOrElse ActionLimits()
-        // This is temporary while we are making sequencing directly supported in the controller.
```

Comment:
Agree, as per discussion yesterday. I will revert the two commits that deal with parameters.